package.json

```diff
@@ -30,13 +30,17 @@
   },
   "dependencies": {
     "prop-types": "^15.6.2",
-    "react-native-animatable": "^1.2.4"
+    "react-native-animatable": "1.3.1"
```

Comment:
I would move this change to a separate PR, since it might have a bigger impact than we expect (so it needs to be tested carefully).

Reply:
This is an error actually, we don't need to upgrade react-native-animatable for this to work so i'm definetely going to revert this 😄

Reply:
@mmazzarolo updated !